html/semantics/forms/form-submission-0/enctypes-helper.js

```diff
@@ -90,12 +126,23 @@
 
   // This function returns a function to add individual form tests corresponding
   // to some enctype.
-  // `expectedBuilder` is a function that takes two parameters: `expected` (the
-  // `expected` value of a test) and `serialized` (the actual form body
-  // submitted by the browser), and returns the correct form body that should
-  // have been submitted. This is necessary in order to account for th
+  // `expectedBuilder` is an optional callback that takes two parameters:
+  // `expected` (the `expected` property passed to a test) and `serialized` (the
+  // actual form body submitted by the browser, isomorphic-decoded). It must
```

Comment:
The example above talks about _actualFormBody, I think serialized here is trying to say the same thing. Why use several terms for the same thing?

Reply:
Right, I'll change that to `actualFormBody` (since the underscore in the example is meant to indicate that the argument is unused).